.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "files.associations": {
```

Comment:
Try not to add this file to the repo if you generate it, it may mess up with mine